src/_css/LoginComponent.css

```diff
@@ -89,7 +89,11 @@ input,
   padding-left: 33%; }
 
 #art-pagination-section {
-  width: 680px; }
+  width: 680px;
+  margin: 0; }
```

Comment:
I have been informed that this is an auto-generated file from sass compiling. Ideally, we should only commit the sass file, excluding the compiled version of it. Ignore for now as I'll look into the set-up later

Reply:
Yes, it's auto-generated as I mentioned in my previous comment.

Reply:
@andela-fmustapha I have been informed that the auto-generated files should not be committed by the people who did the set-up. You can delete them and if they still show up in GitHub, use `git rm --cached <filename>` to delete them